hoverctl/cmd/logs.go

```diff
@@ -19,21 +20,42 @@ Shows the Hoverfly logs.
 	Run: func(cmd *cobra.Command, args []string) {
 		checkTargetAndExit(target)
 
-		logfile := wrapper.NewLogFile(hoverflyDirectory, strconv.Itoa(target.AdminPort), strconv.Itoa(target.ProxyPort))
+		format := "plain"
 
-		if followLogs {
-			err := logfile.Tail()
-			handleIfError(err)
-		} else {
-			err := logfile.Print()
-			handleIfError(err)
+		jsonLogs, _ := cmd.Flags().GetBool("json")
+		if jsonLogs {
+			format = "json"
+		}
+
+		logs, err := wrapper.GetLogs(*target, format)
+		handleIfError(err)
+
+		logsPrinted := map[string]string{
+			"": "x",
+		}
+
+		for i := 0; i < len(logs); i++ {
+
+			if logs[i] != "" && logsPrinted[logs[i]] != "x" {
+				fmt.Println(logs[i])
+				logsPrinted[logs[i]] = "x"
+			}
+
+			if i == len(logs)-1 && followLogs {
```

Comment:
Okay - maybe not as this but as part of another PR, it could have a way of returning diffs. For example providing an `after` parameter, or an `id` of the last retrieved log. Maybe we could create an issue? Because otherwise, the longer it runs, the more network traffic there will be until this call becomes a bottleneck.